projects/vdk-plugins/vdk-duckdb/tests/test_plugin.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+import unittest
+
+import duckdb
+from vdk.internal.core.config import Configuration
+from vdk.internal.core.errors import UserCodeError  #
+from vdk.plugin.duckdb.duckdb_configuration import DuckDBConfiguration
+from vdk.plugin.duckdb.ingest_to_duckdb import IngestToDuckDB
+
+
+class TestIngestToDuckDB(unittest.TestCase):
```

Comment:
You have a lot of `.db` files that you shouldn't add to version control. Please remove them. I'm guessing they're from local test runs you did.